src/Admin/OnboardingWizardSubmenuPage.php

```diff
@@ -255,5 +264,23 @@ public function enqueue_assets( $hook_suffix ) {
 				'after'
 			);
 		}
+
+		$this->add_preload_rest_paths();
+	}
+
+	/**
+	 * Adds REST paths to preload.
+	 */
+	public function add_preload_rest_paths() {
```

Comment:
Should these methods be protected?

```suggestion
	protected function add_preload_rest_paths() {
```